internal/venice-common/src/main/java/com/linkedin/venice/pushstatushelper/PushStatusStoreReader.java

```diff
@@ -101,7 +101,9 @@ public Map<CharSequence, Integer> getPartitionStatus(
       if (pushStatusValue == null) {
         return Collections.emptyMap();
       } else {
-        LOGGER.info(" {}/{} Instance status: {}", storeName, partitionId, pushStatusValue.instances);
+        if (pushStatusValue.instances.size() < 20) {
```

Comment:
IMHO, rather than logging this only when there are fewer than 20 instances, we should log all unique DVC client hostnames where ingestion is still incomplete. In most cases, this is far more useful for troubleshooting than just listing the completed partitions.